### PR TITLE
Fix NEWS.md link to manual section on customizing colors

### PR DESCRIPTION
**Pull request by TotalVerb**
[ci skip]

### CODE REVIEW

**Comment by TotalVerb** [comment]
I should note that I have run

```sh
$ make check-whitespace
```

locally without issue.

**Comment by tkelman** [comment]
the bullet right after this one could use a PR reference

**Comment by TotalVerb** [comment]
I have added PR references for the bullet right after this one.

File: NEWS.md [hunk 1]
```diff
@@ -166,10 +166,12 @@ Library improvements
   * The default text style for REPL input and answers has been changed from bold to normal ([#11250]).
     They can be changed back to bold by setting the environment variables `JULIA_INPUT_COLOR` and `JULIA_ANSWER_COLOR` to `"bold"`.
     For example, one way of doing this is adding `ENV["JULIA_INPUT_COLOR"] = :bold` and `ENV["JULIA_ANSWER_COLOR"] = :bold` to the `.juliarc.jl` file.
-    See the [manual section on customizing colors](http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors) for more information.
+    See the [manual section on customizing colors](http://docs.julialang.org/en/latest/manual/interacting-with-julia#Customizing-Colors-1) for more information.
```
Comment (KristofferC) [inline NEWS.md]: Should this link to the `latest` / `stable` or explicit `0.6` manual?

Comment (TotalVerb) [inline NEWS.md]: As of now, there is no explicit 0.6 manual. This can be revised after release.

**Pull request merged**